s3/pkg/datastore/yig/tests/utils_test.go

```diff
@@ -0,0 +1,56 @@
+package tests
+
+import (
+	"sync"
+
+	"github.com/opensds/multi-cloud/s3/pkg/datastore/yig/utils"
+	. "gopkg.in/check.v1"
+)
+
+func (ys *YigSuite) TestGlobalId(c *C) {
+	gi, err := utils.NewGlobalIdGen()
+	c.Assert(err, Equals, nil)
+	c.Assert(gi, Not(Equals), nil)
+	ids := make(map[int64]int64)
+	count := 8192
+
```

Comment:
why it's 8192, please define a constant with a description

Reply:
thanks @wisererik for your suggestion, 8192 is just a number which contains boundary for the max number of ids per ms. i will define a constant for it.